engine/apps/schedules/ical_utils.py

```diff
@@ -624,77 +651,56 @@ def create_base_icalendar(name: str) -> Calendar:
     return cal
 
 
-def get_events_from_calendars(ical_obj: Calendar, calendars: tuple) -> None:
-    for calendar in calendars:
-        if calendar:
-            for component in calendar.walk():
-                if component.name == "VEVENT":
+def get_user_events_from_calendars(
+    ical_obj: Calendar, calendar: Calendar, user: User, name: typing.Optional[str] = None
+) -> None:
+    if calendar:
+        for component in calendar.walk():
+            if component.name == "VEVENT":
+                event_user = get_usernames_from_ical_event(component)
+                event_user_value = event_user[0][0]
+                if event_user_value == user.username or event_user_value.lower() == user.email.lower():
+                    if name:
+                        component["SUMMARY"] = "{}: {}".format(name, component["SUMMARY"])
                     ical_obj.add_component(component)
 
 
-def get_user_events_from_calendars(ical_obj: Calendar, calendars: tuple, user: User, name: str = None) -> None:
-    for calendar in calendars:
-        if calendar:
-            for component in calendar.walk():
-                if component.name == "VEVENT":
-                    event_user = get_usernames_from_ical_event(component)
-                    event_user_value = event_user[0][0]
-                    if event_user_value == user.username or event_user_value.lower() == user.email.lower():
-                        if name:
-                            component["SUMMARY"] = "{}: {}".format(name, component["SUMMARY"])
-                        ical_obj.add_component(component)
```

Comment:
the `calendars` argument here was **always** being passed in as a `tuple` w/ one `Calendar` object, let's just instead pass in the `Calendar` argument directly instead of needing to loop over one element.